backend/src/test/java/hanglog/expense/domain/AmountTest.java

```diff
@@ -0,0 +1,37 @@
+package hanglog.expense.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import hanglog.global.exception.InvalidDomainException;
+import java.math.BigDecimal;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class AmountTest {
+
+    @DisplayName("비용이 1억 초과이면 예외가 발생한다.")
+    @Test
+    void validateAmount_OverMax() {
+        // given
+        final BigDecimal amountValue = BigDecimal.valueOf(100_000_001);
+
+        // when & then
+        assertThatThrownBy(() -> new Amount(amountValue))
+                .isInstanceOf(InvalidDomainException.class)
+                .extracting("code")
+                .isEqualTo(3008);
```

Comment:
```suggestion
       assertThatThrownBy(() -> new Amount(amountValue))
                .isInstanceOf(InvalidDomainException.class)
                .hasMessage(INVALID_EXPENSE_OVER_MAX.getMessage())
                .extracting("code")
                .isEqualTo(INVALID_EXPENSE_OVER_MAX.getCode());
```

요런식으로 ExceptionCode 가져와서 비교하는 건 어떤가요?

Reply:
좋아요
다른 테스트에서 사용한 방식이랑 통일한건데... 한 번에 고치는 게 낫겠네요
`TripServiceTest`, `CurrencyServiceTest`, `ImageFileTest`